RxExample/RxExample/Examples/GeolocationExample/GeolocationViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  GeolocationViewController.swift
+//  RxExample
+//
+//  Created by Carlos García on 19/01/16.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+#if !RX_NO_MODULE
+    import RxSwift
+    import RxCocoa
+#endif
+
+class GeolocationViewController: ViewController {
+    
+    @IBOutlet weak private var noGeolocationView: UIView!
+    @IBOutlet weak private var button: UIButton!
+    @IBOutlet weak private var button2: UIButton!
+    @IBOutlet weak private var latLabel: UILabel!
+    @IBOutlet weak private var lonLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let geolocationService = GeolocationService.instance
+        
+        unowned let _self = self
```

Comment:
I don't think we should be giving examples with `unowned`. I think it's probably ok in this case, but it's extremely easy to create subtle errors with `unowned` and crash the app.

Maybe it would be better to have instead of
```
private func driveAutorization(autorized: Bool) {
 +        if autorized {
 +            noGeolocationView.hidden = true
 +            view.sendSubviewToBack(noGeolocationView)
 +        }
 +        else {
 +            noGeolocationView.hidden = false
 +            view.bringSubviewToFront(noGeolocationView)
 +        }
 +    }
```
an extension on `UIView`

```
var rx_authorizationState: Bool<Bool> 
```

with that logic.

In that case you can just say `drive(noGeolocationView.rx_authorizationState)`.

The same solution can apply to `driveCoordinates`. We can have only one label with 
```
"Latitude: \(location.latitude)\nLongitude: \(location.longitude)"
```